<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统优化-管家婆</string>
  </property>
  <widget class="SystemoptTitleBar" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back_group/Resources/min_button/catalog_8.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Min">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/back_group/Resources/min_button/minback.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../SaftUI/Resources/min_button/minb1.png</normaloff>../SaftUI/Resources/min_button/minb1.png</iconset>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_close">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/back_group/Resources/min_button/minback.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../SaftUI/Resources/min_button/close.png</normaloff>../SaftUI/Resources/min_button/close.png</iconset>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidgetUI">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>821</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_start">
    <attribute name="title">
     <string>一键优化</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_server">
    <attribute name="title">
     <string>启动项</string>
    </attribute>
    <widget class="QLabel" name="label_tips">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>你的电脑共有10个启动项</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Startitem">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>显示已禁用的项</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>6</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;a href=&quot;www.baidu.com&quot;&gt;刷新&lt;/a&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_RunTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>811</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_tips_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>“启动项”在开机后自动运行，建议关闭不必要的项</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>服务项</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_Service">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>791</width>
       <height>391</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>你的电脑共有48个开机启动服务项，禁用不需要的启动项可以加快开机速度</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>显示被禁止的项</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>计划任务</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_starttime">
    <attribute name="title">
     <string>开机时间管理</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Load_Tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正在扫描开机项......</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Opt_Load_Gif">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SystemoptTitleBar</class>
   <extends>QWidget</extends>
   <header>SystemoptTitleBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../SaftUI/saftui.qrc"/>
 </resources>
 <connections/>
</ui>
